src/core/logic.py

```diff
@@ -1820,3 +1821,46 @@ def get_active_proposal_form():
         active_form = models.ProposalForm.objects.filter(in_edit=False).first()
 
     return active_form
+
+
+def get_file_mimetype(file_path):
+    """
+
+    Args:
```

Comment:
They are not arguments, they are parameters. I don't care what Google says, they are wrong

https://stackoverflow.com/questions/156767/whats-the-difference-between-an-argument-and-a-parameter

Reply:
Interesting concept. 
Would this also mean we should use `*params` and `**kwparams` instead of `*args` and `**kwargs`?

Reply:
No, args are appropriate as a parameter name, but args is not appropriate as the name of what is passed to the method. Params are the variable names, args can be used as a variable name, and that variable name is a param